blocks/form/gdpr.js

```diff
@@ -0,0 +1,168 @@
+const apiClass = 'apiKey';
+const feedbackClass = 'feedback';
+const inputApi = document.getElementById(apiClass);
+const formBlock = document.querySelector('.gdpr');
+const feedback = document.querySelector('.feedback');
+const fbWrapper = feedback.children[0];
+const fbContainer = fbWrapper.children[0];
+const form = formBlock.querySelector('form');
+const inputs = form.querySelectorAll('input');
+const ticket2Link = '/ticket2link?ticket=';
+const urlProd = 'https://api.netcentric.biz/gdpr';
+const urlBase = urlProd;
+
+const savedApiKey = localStorage.getItem(apiClass);
+inputApi.value = savedApiKey ?? '';
+
+fbWrapper.classList.add(`${feedbackClass}-wrapper`);
+fbContainer.classList.add(`${feedbackClass}-container`);
+
+/**
+ * Add a CSS class to an HTMLElement
+ * @param {HTMLElement} el Element to add the className
+ * @param {string} className
+ */
+function addClass(el, className) {
```

Comment:
not sure if it really makes sense to have a function for this instead of calling the classList.add function directly where it is needed